Makefile

```
@@ -66,19 +66,17 @@ INTEG_TEST_NDC_ROOT    := ./host/ndc
 PROTO_ROOT := proto
 PROTO_FILES = $(shell find ./$(PROTO_ROOT)/internal -name "*.proto")
 PROTO_DIRS = $(sort $(dir $(PROTO_FILES)))
-PROTO_IMPORTS := -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(GOPATH)/src/github.com/temporalio/gogo-protobuf/protobuf
+PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell cd build && go list -m -f '{{.Dir}}' github.com/temporalio/gogo-protobuf)/protobuf
```

One more idea. We could do this instead:
```suggestion
PROTO_IMPORTS = -I=$(PROTO_ROOT)/internal -I=$(PROTO_ROOT)/api -I=$(shell go list -modfile go-tools.mod -m -f '{{.Dir}}' github.com/temporalio/gogo-protobuf)/protobuf
```
i.e. add `-modfile go-tools.mod` to each command instead of `cd build &&`, so that we don't need a subdirectory